.travis.yml

```diff
@@ -1,8 +1,17 @@
-language: python
-python:
-  - "3.6"
+language: rust
+
+rust:
+  - stable
+
 install:
-  - pip install pyyaml
+  - wget https://bootstrap.pypa.io/get-pip.py
+  - python3 get-pip.py --user
+  - python3 -m pip install --user pyyaml
```

Comment:
`python3-yaml` is on the Travis apt whitelist, so maybe we could replace this with
```
addons:
  apt:
    packages:
      - python3
```

Reply:
Nice!